agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/telemetry/TelemetryClient.java

```diff
@@ -245,8 +245,8 @@ private BatchItemProcessor initBatchItemProcessor(
               // will log if that retry from disk fails
               new DiagnosticTelemetryPipelineListener(
                   "Sending telemetry to the ingestion service",
-                  true,
-                  " (telemetry will be stored to disk and retried)"),
+                  false,
```

Comment:
not get it why it's false here.

Reply:
so that we only log the 2nd time a batch fails (when retrying from disk)